datadog_checks_base/datadog_checks/base/checks/windows/perf_counters/base.py

```diff
@@ -25,6 +30,10 @@ class PerfCountersBaseCheck(AgentCheck):
     def __init__(self, name, init_config, instances):
         super().__init__(name, init_config, instances)
 
+        self.interval = 60
+        if datadog_agent and datadog_agent.get_config('windows_counter_refresh_interval'):
```

Comment:
Do we need to document it in some wayh

Reply:
Yes, we will add the documentation for this feature and the nuances once this feature is in.